superset/assets/src/explore/store.js

```diff
@@ -56,6 +56,10 @@ export function getControlsState(state, form_data) {
       delete control.mapStateToProps;
     }
 
+    if (control.multi && typeof formData[k] === 'string') {
```

Comment:
nit: `formData[k] = (control.multi && typeof formData[k] === 'string') ? [formData[k]] : formData[k]`

Reply:
I think we'd want to stick with `typeof formData[k] === 'string'` because formData[k] might be undefined or null and we would want to leave it as null.

Reply:
Though I think it can be a `int` / `float` too.

Reply:
Ah ok, updated it to `formData[k] && !Array.isArray(formData[k])`.